.gitignore

```diff
@@ -8,5 +8,6 @@
 /website/node_modules
 npm-debug.log
 build
-website/core/metadata.js
+website/core/metadata*.js
 website/src/jest/docs
+website/src/jest/blog/2*
```

Comment:
The way react-page-middleware is setup is that it takes all the js files inside of src/, runs them and outputs html for them. There is no concept of router.

So, there are some files that are manually written like src/jest/index.js and some that are generated like src/jest/blog/2015/05/02/blog.js. We need to make sure that we don't check in those generated files.

The system is a bit ghetto but it works :)